torch/csrc/distributed/c10d/init.cpp

```diff
@@ -31,14 +32,21 @@
 #include <c10d/frontend.hpp>
 #include <c10d/logger.hpp>
 #include <c10d/reducer.hpp>
+
 #include <torch/csrc/Exceptions.h>
 #include <torch/csrc/distributed/c10d/python_comm_hook.h>
+#include <torch/csrc/distributed/c10d/quantization/quantization.h>
 #include <torch/csrc/jit/python/pybind_utils.h>
 #include <torch/csrc/utils/object_ptr.h>
 #include <torch/csrc/utils/pybind.h>
 
 #include <torch/custom_class.h>
 
+#include <ATen/ATen.h>
+#include <ATen/TypeDefault.h>
```

Comment:
I don't think any of these four includes are needed, maybe <ATen/ATen.h> (although I don't think that's needed as well).